.gitignore

```diff
@@ -9,6 +9,7 @@
 /.bundle
 /dist/
 /doc/
+/storage/
```

Comment:
This gitignore if the rails repository gitignore not the application's gitignore. You want to change this one.

Reply:
`/storage/*` is already there.

Reply:
Perhaps @murb upgraded a Rails app to 5.2 and thus didn't get the `.gitignore` line added?

Reply:
Was indeed in an upgraded project; sorry.